lib/connect.js

```diff
@@ -375,8 +393,38 @@ const LogLevel = {
   off: 'off'
 };
 
-function createLocalParticipant(signaling, log, encodingParameters, options, localTracks) {
-  const localParticipantSignaling = signaling.createLocalParticipantSignaling(encodingParameters);
+/**
+ * The verbosity level of network quality information of a {@link Participant}.
+ * @enum {number}
+ */
+// eslint-disable-next-line
+const NetworkQualityVerbosity = {
+  /**
+   * Nothing is reported for the {@link Participant}. This has no effect and
+   * defaults to {@link NetworkQualityVerbosity#minimal} for the {@link LocalParticipant}.
+   */
+  none: 0,
+  /**
+   * Reports {@link NetworkQualityLevel} for the {@link Participant}.
+   */
+  minimal: 1,
+  /**
+   * Reports {@link NetworkQualityLevel} and {@link NetworkQualityStats} for the {@link Participant}.
+   * {@link NetworkQualityStats} is populated with audio and video {@link NetworkQualityLevel}s
+   * based on which the {@link Participant}'s {@link NetworkQualityLevel} is calculated.
+   */
+  moderate: 2,
+  /**
+   * Reports {@link NetworkQualityLevel} and {@link NetworkQualityStats} for the participant.
```

Comment:
Maybe replace participant with "{@link Participant}"

Could also say something like, "in addition to the reported statistics reported for minimal, the corresponding {@link NetworkQualityMediaStats} is also reported.

Reply:
@aymenn ,

We already say this with the following phrase:

> Reports **{@link NetworkQualityLevel}** and {@link NetworkQualityStats}